kubecost/templates/aggregator/aggregator-statefulset.yaml

```diff
@@ -399,7 +399,7 @@ spec:
               mountPath: /var/configs/actions/storage
             - name: federated-storage-config
               mountPath: /var/configs/actions/storage/actions-store.yaml
-              subPath: federated-store.yaml
+              subPath: actions-store.yaml
```

Comment:
I don't really get what we're doing here. We're mounting the actions store to the federated storage? Are both the actions-store.yaml and federated-store.yaml the same file but renamed?

Reply:
Ok makes sense!